src/main/java/edu/usf/cutr/gtfsrtvalidator/background/BackgroundTask.java

```diff
@@ -121,6 +121,10 @@ public void run() {
                 }
                 session.save(feedIteration);
                 GTFSDB.commitAndCloseSession(session);
+
+                if(!isUniqueFeed) {
```

Comment:
Minor formatting issue - can you please reformat as `if (!isUniqueFeed) {`?

It's missing a space after `if`.